### PR TITLE
Sdk420

### PR DESCRIPTION
**Pull request by matemax**
support sdk420

### CODE REVIEW

File: lunavl/sdk/faceengine/setting_provider.py [hunk 1]
```diff
@@ -544,13 +544,15 @@ class FaceDetV3Settings(BaseSettingsSection):
...
-        - imageSize (int): Target image size for down scaling by load side;
+        - minFaceSize (int): Minimum face size in pixels;
+        - maxFaceSize (int): Maximum face size in pixels;
         - nms (NMS): type of NMS: mean or best;
         - redetectTensorSize (int): target face after preprocessing for redetect;
         - redetectFaceTargetSize (int): target face size for redetect;
         - paddings (Point4): paddings;
         - paddingsIR (Point4): paddingsIR;
         - planPrefix (str): planPrefix;
+        - useOrientationMode (bool): Usemode for rotated origin images or not;
```
Comment (matemax) [inline lunavl/sdk/faceengine/setting_provider.py]: .

File: lunavl/sdk/faceengine/setting_provider.py [hunk 2]
```diff
@@ -613,7 +615,26 @@ def NMSThreshold(self, value: float) -> None:
...
+        Setter for minFaceSize
+        Args:
+            value: new value
+        """
+        self.setValue("minFaceSize", value)
+
+    @property
+    def maxFaceSize(self) -> int:
         """
         Getter for imageSize
```
Comment (matemax) [inline lunavl/sdk/faceengine/setting_provider.py]: .

File: lunavl/sdk/faceengine/setting_provider.py [hunk 3]
```diff
@@ -911,6 +970,181 @@ class FaceDetV2Settings(FaceDetV12Settings):
...
+        - NMSThreshold (float): overlap threshold for NMS [0..1] range;
+        - minFaceSize (int): Minimum face size in pixels;
+        - maxFaceSize (int): Maximum face size in pixels;
+        - nms (NMS): type of NMS: mean or best;
+        - redetectTensorSize (int): target face after preprocessing for redetect;
+        - redetectFaceTargetSize (int): target face size for redetect;
+        - paddings (Point4): paddings;
+        - paddingsIR (Point4): paddingsIR;
+        - planPrefix (str): planPrefix;
+        - useOrientationMode (bool): Usemode for rotated origin images or not;
```
Comment (matemax) [inline lunavl/sdk/faceengine/setting_provider.py]: .

File: lunavl/sdk/estimators/face_estimators/mask.py [hunk 4]
```diff
@@ -33,62 +67,70 @@ def __init__(self, mask: MedicalMaskEstimation):
...
         """
         Convert to dict.
 
         Returns:
             {
-                "mask_in_place": self.maskInPlace,
-                "mask_not_in_place": self.maskNotInPlace,
-                "no_mask": self.noMask,
-                "occluded_face": self.occludedFace,
+            "predominant_emotion": predominantName,
```
Comment (gluhar2006) [inline lunavl/sdk/estimators/face_estimators/mask.py]: not emotion

File: tests/schemas.py [hunk 5]
```diff
@@ -64,14 +64,18 @@
...
     "type": "object",
     "properties": {
-        "mask_in_place": TYPE_SCORE,
-        "mask_not_in_place": TYPE_SCORE,
-        "no_mask": TYPE_SCORE,
-        "occluded_face": TYPE_SCORE,
+        "estimations": MASK_ESTIMATION_SCHEMA,
+        "predominant_mask": {"type": "string", "enum": ["medical_mask", "missing", "occluded"]},
     },
-    "additionalProperties": False,
```
Comment (gluhar2006) [inline tests/schemas.py:75]: revert additionalProperties

File: tests/test_mask.py [hunk 6]
```diff
@@ -55,39 +54,33 @@ def test_estimate_mask_as_dict(self):
...
+        mask = TestMask.maskEstimator.estimate(self.warpImageMissing)
         self.assertMaskEstimation(mask, expectedResult._asdict())
 
-    def test_estimate_mask_not_in_place(self):
-        """
-        Test mask estimations with mask exists on the face and is not worn properly
-        """
-        expectedResult = MaskProperties(0.042, 0.386, 0.003, 0.567)
-        mask = TestMask.maskEstimator.estimate(self.warpImageMaskNotInPlace)
         self.assertMaskEstimation(mask, expectedResult._asdict())
```
Comment (gluhar2006) [inline tests/test_mask.py]: now it's unnecessary

File: lunavl/sdk/estimators/face_estimators/mask.py (hunk 4 above)
Comment (matemax) [reply lunavl/sdk/estimators/face_estimators/mask.py]: +

File: tests/test_mask.py (hunk 6 above)
Comment (matemax) [reply tests/test_mask.py]: +

File: tests/schemas.py (hunk 5 above)
Comment (matemax) [reply tests/schemas.py:75]: +

**Pull request merged**